src/sbt-test/sbt-conductr/sandbox-conductr-roles/test

```diff
@@ -4,7 +4,7 @@
 
 > sandbox stop
 
-> sandbox run 2.0.0 --nr-of-containers 4 --conductr-role new-role --conductr-role other-role
+> sandbox run 2.0.0 --nr-of-containers 4 --conductr-role new-role haproxy --conductr-role other-role haproxy
```

Comment:
sandbox wouldn't ever start up because haproxy wouldn't get a resource offer due to missing role